clients/unshaded/src/main/java/tachyon/client/table/AbstractTachyonRawTables.java

```diff
@@ -30,10 +29,10 @@
  */
 @PublicApi
 public abstract class AbstractTachyonRawTables implements TachyonRawTablesCore {
-  protected RawTablesContext mContext;
+  protected RawTableContext mContext;
```

Comment:
I think the context was named to be consistent with the interface `TachyonRawTablesCore`? @calvinjia